CMakeLists.txt

```diff
@@ -86,6 +86,10 @@ if(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
     "MinSizeRel" "RelWithDebInfo")
 endif()
 
```

Comment:
PS:  This is the mechansims used by LAPACK, ITK, VTK, QT, and many many other cmake based packages.